File: src/main/java/club/xiaojiawei/entity/CommonEntity.java
package club.xiaojiawei.entity;

import club.xiaojiawei.enums.ZoneEnum;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

/**
 * @author 肖嘉威
 * @date 2022/11/30 12:38
 */
@EqualsAndHashCode(callSuper = true)
@Data
@ToString(callSuper = true)
public class CommonEntity extends Entity{

    private String entity;

    private ZoneEnum zone;

    private int zonePos;

    private String playerId;

}

File: Hearthstone-Script/src/main/java/club/xiaojiawei/bean/log/CommonEntity.kt
package club.xiaojiawei.bean.log

import club.xiaojiawei.bean.Entity
import club.xiaojiawei.enums.ZoneEnum
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.ToString

/**
 * @author 肖嘉威
 * @date 2022/11/30 12:38
 */

open class CommonEntity : Entity() {

    var entity: String = ""

    var zone: ZoneEnum? = null

    var zonePos = 0

    var playerId: String = ""

    var logType: String = ""

}
